log.go

```diff
@@ -41,6 +43,9 @@ type StandardLogger interface {
 type EventLogger interface {
 	StandardLogger
 
+	Warn(args ...interface{})
```

Comment:
Why did we move these?

Reply:
I can't require them on StandardLogger as `why/log` doesn't implement them.